backend/app/views/spree/admin/roles/edit.html.erb

```diff
@@ -6,6 +6,8 @@
   <%= back_to_list_button(Spree::Role.model_name.human, admin_roles_path) %>
 <% end %>
 
+<%= render :partial => 'spree/admin/shared/error_messages', :locals => { :target => @role } %>
```

Comment:
@damianlegawiec OK. Lets leave it for now and change hash syntax as per new ruby in another PR.